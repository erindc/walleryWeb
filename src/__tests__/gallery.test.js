import Gallery from '../pages/GalleryPage';
import React from 'react'
import { shallow } from 'enzyme'

describe('Gallery', () => {
  it('renders without crashing', () => {
    const component = shallow(<Gallery />)
    expect(component).toMatchSnapshot()
  })
})