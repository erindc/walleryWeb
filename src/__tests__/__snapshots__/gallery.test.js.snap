// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery renders without crashing 1`] = `
<Route>
  <Component />
</Route>
`;
